frontend/src/scenes/settings/organization/Members.tsx

```diff
@@ -69,11 +69,13 @@ function ActionsComponent(_: any, member: OrganizationMemberType): JSX.Element |
                                     }
                                     if (listLevel === OrganizationMembershipLevel.Owner) {
                                         LemonDialog.open({
-                                            title: `Transfer organization ownership to ${fullName(member.user)}?`,
-                                            description: `You will no longer be the owner of ${user.organization?.name}. After the transfer you will become an administrator.`,
+                                            title: `Add additional owner to ${user.organization?.name}?`,
+                                            description: `Please confirm that you would like to make ${fullName(
+                                                member.user
+                                            )} an owner of ${user.organization?.name}.`,
                                             primaryButton: {
                                                 status: 'danger',
-                                                children: 'Transfer Ownership',
+                                                children: `Make ${fullName(member.user)} owner`,
```

Comment:
Does it sound better as `Make ${fullName(member.user)} an owner`?

Reply:
Yes, good point.

Reply:
Didn't see this get updated